src/tests/certify.rs

```diff
@@ -365,6 +365,173 @@ fn mock_prune_non_importable_audit() {
     insta::assert_snapshot!("mock-prune-non-importable-audit", result);
 }
 
+#[test]
+fn mock_collapse_non_importable_audits() {
```

Comment:
Can we have a test which collapses with a full audit to a git revision (e.g. `1.0.0@git:` plus `1.0.0@git: -> 1.0.0`?)

Having a test which doesn't merge with a prior delta audit which doesn't validate for all criteria specified would also be nice.

Reply:
We have a test for a prior delta audit which doesn't validate for the criteria specified (`mock_dont_collapse_incompatible_criteria_non_importable_audits`), unless you mean something else?